src/Features/Lsif/Generator/Writing/LsifConverter.cs

```diff
@@ -12,8 +13,18 @@ internal sealed class LsifConverter : JsonConverter
     {
         public override bool CanConvert(Type objectType)
         {
-            return typeof(ISerializableId).IsAssignableFrom(objectType) ||
-                   objectType == typeof(Uri);
+            if (typeof(ISerializableId).IsAssignableFrom(objectType) ||
+                objectType == typeof(Uri))
+            {
+                return true;
+            }
+
+            if (objectType.IsConstructedGenericType && objectType.GetGenericTypeDefinition() == typeof(Nullable<>))
```

Comment:
why do we need special handling of nullable?  Does newtonsoft not handle it appropriately?

Reply:
@dibarbet For whatever reason, it did not. ¯\_(ツ)_/¯ I didn't really investigate, because moving this to System.Text.Json will probably happen soonish.